packages/ui-library/src/components/forms/text-input/index.ts

```diff
@@ -131,18 +131,20 @@ export class BlrTextInput extends LitElement {
           ${dynamicStyles}
         </style>
         <div class="blr-text-input ${classes}">
-          ${this.hasLabel
-            ? html`
-                ${BlrFormLabelRenderFunction({
-                  labelText: this.label,
-                  labelSize: this.size,
-                  labelAppendix: this.labelAppendix,
-                  forValue: this.textInputId,
-                  theme: this.theme,
-                  variant: this.hasError ? 'error' : 'label',
-                })}
-              `
-            : nothing}
+          <div class="label-wrapper">
+            ${this.hasLabel
+              ? html`
```

Comment:
is html thingy required here because in this file 'packages/ui-library/src/components/forms/select/index.ts' in line number 161, same logic exist but with the html part.